rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -171,6 +174,17 @@ class CallbackGroup
   bool
   automatically_add_to_executor_with_node() const;
 
+  /// Create and return the notify guard condition.
+  RCLCPP_PUBLIC
+  std::shared_ptr<rclcpp::GuardCondition>
+  create_notify_guard_condition(
+    const rclcpp::Context::SharedPtr context_ptr);
```

Comment:
nitpick:

```suggestion
  create_notify_guard_condition(const rclcpp::Context::SharedPtr context_ptr);
```

Reply:
I think you asked me to remove the `const` as the constructor of `rclcpp::GuardCondition` uses a non-const context.